dts/riscv/ite/it8xxx2.dtsi

```diff
@@ -860,6 +860,18 @@
 			interrupt-parent = <&intc>;
 			status = "okay";
 		};
+		shi0: shi@f03a00 {
+			compatible = "ite,it8xxx2-shi";
+			reg = <0x00f03a00 0x30>;
+			interrupts = <171 0>;
+			interrupt-parent = <&intc>;
+			pinctrl-0 = <&shi_mosi_gpm0_default
+				     &shi_miso_gpm1_default
+				     &shi_clk_gpm4_default
+				     &shi_cs_gpm5_default>;
+			pinctrl-names = "default";
```

Comment:
I think these should not be in this file and go in the boards using it instead.

Reply:
Shouldn't the default interfaces be defined in the chip device tree?

Reply:
The basic device node yes (it's an intrinsic characteristic of the SoC) but the pin mapping is a board  design decision so it should go in the board file.

Reply:
Had moved this to the EC boards dts, forgot to resolve the conversation.